src/lib/components/FileDropzone/FileDropzone.svelte

```diff
@@ -81,13 +83,7 @@
 				<div class="text-4xl flex justify-center items-center" bind:this={elemIcon}>&darr;</div>
 				<!-- Text -->
 				<div class="flex flex-col justify-center items-start space-y-0">
-					<div class="text-base font-bold">
-						{#if title}
-							{@html title}
-						{:else}
-							Drop {$$restProps.multiple ? 'files' : 'a file'} or click to select.
-						{/if}
-					</div>
+					<div class="text-base font-bold">{@html title}</div>
```

Comment:
My first thought was to implement it that way,  but then I thought it would take away the option for the multiple in the default title. 

Question: What is the @type {string} good for in the comment?

Reply:
This document will explain the use of the JSDocs style comments and typing:
https://www.skeleton.dev/docs/contributions/documentation

I was testing something that would have needed the type defined, but I went another direction, so technically that could have been reverted. It doesn't hurt anything though, so we can leave it be for now.